docs/security/host-certs.md

```diff
@@ -162,15 +165,16 @@ Once you have RA privileges, you may request, approve, and retrieve host certifi
                     --pkey ~/path_to_dir/incommon_user_key.pem \
                     --hostname <HOSTNAME> \
                     --altname <ALTNAME> \
-                    --altname <ALTNAME2>
+                    --altname <ALTNAME2> [--orgcode <ORG,DEPT>]
```

Comment:
Could we put the `--orgcode` on a newline to mimic the formatting of the other options?